crates/host/src/wasmbus/mod.rs

```diff
@@ -1622,12 +1668,12 @@ async fn create_lattice_metadata_bucket(
         })
         .await
     {
-        Ok(_) => {
-            info!(%bucket, "created lattice metadata bucket with 1 replica");
-            Ok(())
+        Ok(store) => {
+            info!("created {bucket_message_name} bucket with 1 replica");
```

Comment:
Due to how log ingestion querying (and billing) tends to work, in general we should emit log messages that have a low cardinality, and put dynamic data in separate fields

Reply:
So I did consider that here. All this is actually doing is making sure the message is correct. It is a little awkward yes, but almost all of the other logic is the same, so this only ends up with 2 messages. However, I think just logging the bucket name is probably good enough at this point